### PR TITLE
Lift

### PR DESCRIPTION
**Pull request by tahsina13**

### CODE REVIEW

File: src/org/usfirst/frc/team694/robot/RobotMap.java [hunk 1]
```diff
@@ -31,17 +31,28 @@
     int RIGHT_MIDDLE_MOTOR_PORT = 5;
     int RIGHT_REAR_MOTOR_PORT = 6;
     /***************************************************************************
-     * Encoder Ports
+     *Drivetrain Encoder Ports
      */
     
     int LEFT_ENCODER_CHANNEL_A = 0;
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/RobotMap.java:37]: Try to be more specific: Something like DRIVETRAIN_LEFT_ENCODER_CHANNEL_A

File: src/org/usfirst/frc/team694/robot/RobotMap.java [hunk 2]
```diff
@@ -31,17 +31,28 @@
...
     int LEFT_ENCODER_CHANNEL_B = 1;
     
     int RIGHT_ENCODER_CHANNEL_A = 2;
     int RIGHT_ENCODER_CHANNEL_B = 3;
+    
     /****************************************************************************
      * Solenoid Ports
      */
     int GEAR_SHIFT_CHANNEL = -1;
+    int LIFT_BRAKE_CHANNEL = -2;
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/RobotMap.java]: In the name, indicate that it is a solenoid port: LIFT_BRAKE_SOLENOID_CHANNEL

File: src/org/usfirst/frc/team694/robot/commands/LiftDownCommand.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.goDown();
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/commands/LiftDownCommand.java]: If isFinished returns true, execute never runs. Move this elsewhere

Pro Tip: For commands that only run once, look at [InstantCommand](http://first.wpi.edu/FRC/roborio/release/docs/java/)

Comment (TacoTechnica) [reply src/org/usfirst/frc/team694/robot/commands/LiftDownCommand.java]: Also, you don't disable your Lift's break before running it. Take a look at your other command where you call Robot.lift.setBrakeOff()

File: src/org/usfirst/frc/team694/robot/commands/LiftStopCommand.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.stop();
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/commands/LiftStopCommand.java]: Same problem as LiftDownCommand: Execute never runs when isFinished returns true. Consider using InstantCommand

File: src/org/usfirst/frc/team694/robot/commands/LiftUpCommand.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        // eg. requires(chassis);
+    }
+
+    // Called just before this Command runs the first time
+    protected void initialize() {
+    }
+
+    // Called repeatedly when this Command is scheduled to run
+    protected void execute() {
+        Robot.lift.goUp();
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/commands/LiftUpCommand.java]: Same problem as LiftDownCommand and LiftStopCommand: Execute never runs when isFinished returns true. Consider using InstantCommand

**Pull request closed without merging**